pkgs/development/libraries/gtk3-nocsd/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, gtk3
+, gobject-introspection
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gtk3-nocsd";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "ZaWertun";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "035rrn9jq9bdfkmmj6xl4q8paqx7xf3hxsw6gslgk86sh7x56lvi";
+  };
+
+  nativeBuildInputs = [
```

Comment:
`pkgconfig` should be `native` but the rest should be `buildInputs` because we actually link against them.

Reply:
Applied